src/components/CrossDataSection.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext } from 'react';
+import { useQuery } from 'react-apollo';
+import { ActivityIndicator } from 'react-native';
+import { NavigationScreenProp } from 'react-navigation';
+
+import { colors, texts } from '../config';
+import { getTitleForQuery, graphqlFetchPolicy } from '../helpers';
+import { useRefreshTime } from '../hooks';
+import { NetworkContext } from '../NetworkProvider';
+import { getQuery, QUERY_TYPES } from '../queries';
+import { DataListSection } from './DataListSection';
+import { LoadingContainer } from './LoadingContainer';
+
+type Props = {
+  dataProviderId: string;
+  dataProviderName: string;
+  navigation: NavigationScreenProp<never>;
+};
+
+const crossDataTypes = [
+  QUERY_TYPES.NEWS_ITEMS,
+  QUERY_TYPES.POINTS_OF_INTEREST,
+  QUERY_TYPES.TOURS,
+  QUERY_TYPES.EVENT_RECORDS
+];
+
+const getNavigationFunction = (
+  navigation: NavigationScreenProp<never>,
+  dataProviderName: string,
+  query: string
+) => {
+  return () =>
+    navigation.push('Index', {
+      queryVariables: { dataProvider: dataProviderName },
+      query,
+      title: getTitleForQuery(query)
+    });
```

Comment:
the category filter does not correspond to the data visible on the index screen. for filtered cross data, the dropdown header seems to need adjustments.

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-03 at 19 45 14-side](https://user-images.githubusercontent.com/1942953/109855908-2636ef00-7c59-11eb-91f0-cc038a9bb412.png)

there are more categories to filter than categories from entries available on the screen.

Reply:
ok

Reply:
ok, so we will wait until somebody asks for it